### ISSUE TITLE
DeepAR covariates scaling support 

### ISSUE THREAD

**Issue by YRChen1998** [awslabs/gluon-ts#2413]
## Description
I read the code of DeepAR. It seems that the model only support to scale the target value. I wonder why not to scale the covariates especially when the covariates are a dynamic real value. (The DeepAR paper standardizes all covariates to have zero mean and unit variance)

**Comment by kashif** [comment]
I believe the target scaling is done as a means to learn the distribution easily in the neural network setting and the scale is carried around and un-done at sampling time... the scale of the covariates does not need to be undone and thus one can scale the covariates as you please in the dataset preparation phase...

**Comment by TD2020lilly** [comment]
I've very interesting findings from the 'scaling' option of DeepAREstimator. What I've found out is if I put 'scaling=False', the prediction output is coming at a different range compared to the input target range. Does anyone has any idea whether we should change anything at prediction side if at the input side 'scaling=False'

**Comment by kashif** [comment]
yes so depends on your datasets... neural networks have a hard time outputting large numbers
